Tasks/GooglePlayHalt/task.loc.json

```diff
@@ -0,0 +1,113 @@
+{
+  "id": "10dada0e-569d-4354-b1c1-7152e6bcdb72",
+  "name": "GooglePlayHalt",
+  "friendlyName": "ms-resource:loc.friendlyName",
+  "description": "ms-resource:loc.description",
+  "author": "Microsoft Corporation",
+  "category": "Deploy",
+  "visibility": [
+    "Build",
+    "Release"
+  ],
+  "demands": [
+    "npm"
+  ],
+  "version": {
+    "Major": "2",
+    "Minor": "190",
```

Comment:
Nit: please don't forget to align it to 191 also

Reply:
Sorry, it has been modified